salt/fileserver/gitfs.py

```diff
@@ -50,7 +50,7 @@
 
 PER_REMOTE_OVERRIDES = ('base', 'mountpoint', 'root', 'ssl_verify',
                         'env_whitelist', 'env_blacklist', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'saltenv')
+PER_REMOTE_ONLY = ('formula', 'name', 'saltenv')
```

Comment:
Oh, interesting. I've never tried using ``global`` as a class attribute, so I've never run into this syntax error before. I'll try to brainstorm some ideas as well.